src/GGTestUtils.py

```diff
@@ -37,7 +37,7 @@ class GGTestUtils:
     _ggComponentToDeleteArn: List[str]
     _ggS3ObjToDelete: List[str]
     _ggServiceList: List[str]
-    _ggDeploymentList: List[str]
```

Comment:
Maybe rename this now as the data it is storing is changing.

`_ggDeploymentToThingNameList` maybe?